src/Visualization/Python/RenderBBH.py

```diff
@@ -0,0 +1,250 @@
+    generate-xdmf. For files to be read properly, the XMF files being pointed to
+    should be in the same directory as the VolumeData.h5 and SurfacesData.h5
+    files.
+
+    To splice all the pictures into a video, try using FFmpeg"""
+    import paraview.simple as pv
+
+    version = pv.GetParaViewVersion()
+    if version < (5, 11) or version > (5, 11):
+        print(
+            "WARNING: Your Paraview version is not 5.11, "
+            "the script may not work correctly."
+        )
+
+    # Volume Data Visualization
+    volume_files_xmf = pv.XDMFReader(
+        registrationName=volume_xmf, FileNames=[volume_xmf]
+    )
+
+    # Check for Lapse and SpatialRicciScalar
+    variables = volume_files_xmf.PointData.keys()
+    lapse_flag = False
+    ricci_scalar_flag = False
+    for x in variables:
+        if x == "Lapse":
+            lapse_flag = True
+        if x == "SpatialRicciScalar":
+            ricci_scalar_flag = True
+    if not (lapse_flag and ricci_scalar_flag):
+        print(
+            "The volume data doesn't have the Lapse or SpatialRicciScalar "
+            "output, the script will not work correctly without those."
+        )
+        return
+
+    render_view = pv.GetActiveViewOrCreate("RenderView")
+
+    # Color the grid
+    color_transfer_function = pv.GetColorTransferFunction("Lapse")
+    opacity_function = pv.GetOpacityTransferFunction("Lapse")
+    color_transfer_function.Discretize = 0
+    color_transfer_function.ApplyPreset(color_map, True)
+    color_transfer_function.InvertTransferFunction()
+
+    # Slice volume data
+    slice = pv.Slice(registrationName="slice", Input=volume_files_xmf)
+    slice.SliceType = "Plane"
+    slice.HyperTreeGridSlicer = "Plane"
+    slice.SliceOffsetValues = [0.0]
+    slice.SliceType.Normal = [0.0, 0.0, 1.0]
+    slice.Triangulatetheslice = 0
+
+    # Warp grid by spatial ricci scalar
+    warp_by_scalar = pv.WarpByScalar(
+        registrationName="WarpByScalar", Input=slice
+    )
+    warp_by_scalar.Scalars = ["POINTS", "SpatialRicciScalar"]
+    warp_by_scalar.ScaleFactor = 2.5
+    warp_by_scalar.Normal = [0.0, 0.0, -1.0]
+    warp_by_scalar_display = pv.Show(
+        warp_by_scalar, render_view, "GeometryRepresentation"
+    )
+    warp_by_scalar_display.SetScalarBarVisibility(render_view, False)
+
+    # Apparent Horizon Visualization
+    if aha_xmf:
+        ah_vis(aha_xmf, render_view)
+    if ahb_xmf:
+        ah_vis(ahb_xmf, render_view)
+
+    if show_grid:
+        warp_by_scalar_display.Representation = "Surface With Edges"
+
+    pv.LoadPalette(paletteName="GradientBackground")
+    render_view.OrientationAxesVisibility = 0
+    pv.SetActiveSource(warp_by_scalar)
+    warp_by_scalar_display.Opacity = 0.8
+    pv.ColorBy(warp_by_scalar_display, ("POINTS", "Lapse"))
+    layout = pv.GetLayout()
+    layout.SetSize(1920, 1080)
+
+    # Camera placements
+    # Top down view
+    if camera_angle == "1":
```

Comment:
I like this idea of good predefined camera angles. Use names instead of numbers though.